Dockerfile

```diff
@@ -11,18 +11,17 @@ ENV DEBIAN_FRONTEND="noninteractive" \
     BOINC_CMD_LINE_OPTIONS=""
 
 # Install
-RUN apt update && apt install -y --no-install-recommends \
-    boinc-client \
- && apt clean \
- && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        boinc-client \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
I checked and removing apt-clean did not affect the container size, and there were no files in `/tmp` or `/var/tmp`. I also switched `apt` -> `apt-get` to get rid of the warning.

Reply:
Good point, this is the official recommendation by docker.
>The official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.